aaws.sh

```diff
@@ -41,6 +41,15 @@ function aaws()
     return 0
   fi
 
+  if [ "-l" == "$@" ]; then
+    printf "Available Profiles:"
+    for e in ${AAWS_AUTOCOMPLETE[@]}; do
+      printf " $e"
+    done
+    printf "\n"
+    return 0
+  fi
```

Comment:
```suggestion
  if [ "-l" == "$@" ]; then
    printf "Available Profiles:\n"
    for e in ${AAWS_AUTOCOMPLETE[@]}; do
      printf "  $e\n"
    done
    return 0
  fi
```

I have a long list of profiles and it is nicer (IMO 😄 ) to see them listed vertically that going out to the side

Looks something like this now

```
$ aaws -l
Available Profiles:
  a
  b
  c
```

Reply:
I added help, changed to vertical and removed the "Available Profiles:" text because I think it is cleaner with vertical listing.

Reply:
I pushed the changes to the forked branch. Not sure what is next for me.